### PR TITLE
Add a formatter, and format all code files

### PR DESCRIPTION
**Pull request by oleorhagen**
This is a follow up of the old Format PR (Note to self, link to it tomorrow).

This adds a CI job, which verifies the formatting

And: formats all the source code files with according to the new formatting rules.

### CODE REVIEW

**Comment by oleorhagen** [comment]
Follow up of: https://github.com/mendersoftware/mender-convert/pull/299

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -52,6 +52,53 @@ build:
...
+  before_script:
+    - apk update
+    - apk upgrade
+    - apk add --update bash git shfmt
+    - shfmt -version
+  script:
+    - /bin/bash
+    - SCRIPTS=$(find modules configs scripts -type f
+            -not -name "*.md"
+            -not -wholename "scripts/test/*")
```
Comment (lluiscampos) [inline .gitlab-ci.yml:68]: I haven't seen this syntax before. Does this work as expected? Opens `bash` and then runs the rest of the script?

**Review by lluiscampos — APPROVED** [review]
Nice! Thanks for following-up, :medal_sports: for you

File: .gitlab-ci.yml (hunk 1 above)
Comment (oleorhagen) [reply .gitlab-ci.yml:68]: 1. Works as expected
2. Surprised me too
3. We could stick to `sh`, but I like `Bash` more.

Comment (lluiscampos) [reply .gitlab-ci.yml:68]: No, this is very nice to know. I don't remember where, but we have used some weird hacks to run things in bash.

Actually, it kind of makes sense. It just runs a `bash` process, and then gitlab keeps piping to/from stdin/stdout.

Comment (oleorhagen) [reply .gitlab-ci.yml:68]: haha, yup. It actually makes kinda sense :)

**Pull request merged**